inc/zoo/swar/SWAR.h

```diff
@@ -70,6 +71,19 @@ struct SWAR {
 
     constexpr T value() const noexcept { return m_v; }
 
+   constexpr static T baseFromLaneLiterals(std::initializer_list<T> args) noexcept {
```

Comment:
@thecppzoo i wonder what you think about this utility?

Reply:
yeah i was trying a few variants with variadic arguments but that was a little tricky when trying to iterate over them, i feel like there is yet a better solution. 

this is one i was inspired for right now, but i think i will defo have more ideas with respect to ergonomics!

Reply:
@thecppzoo how does this update look? makes it unambiguous that the number of lanes to use. 
how would you go about using the different types for the arguments here dynamically?